.all-contributorsrc

```diff
@@ -1650,6 +1650,17 @@
       ]
     },
     {
+ 
+      "login": "OluwabukolaU",
+      "name": "Sophia Umaru",
+      "avatar_url": "https://avatars.githubusercontent.com/u/111010273?v=4",
+      "profile": "https://github.com/OluwabukolaU",
+      "contributions": [
+        "question",
+        "a11y",
+        "tutorial",
+        "userTesting"
+
       "login": "nickaldwin",
```

Comment:
@adiati98, I think the duplication error lies here. It's missing an opening bracket {

Reply:
Yep. And the closing brackets.

Reply:
You can add them manually. And make sure to test it locally before pushing the changes so you know if it breaks the production or not.

Then, create a new screenshot of the README and, in this case, the badge to reflect the changes👍

I believe you got these when resolving the merge conflicts. You can read and follow the instruction on [how to resolve merge conflicts in guestbook repo](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository) on our intro course.